admin/server-timing.php

```diff
@@ -58,25 +99,23 @@ static function() {
 					),
 					array( 'code' => array() )
 				);
-				?>
-				<br>
-				<?php
 				echo wp_kses(
 					__( 'For any hook name provided, the <strong>cumulative duration between all callbacks</strong> attached to the hook is measured, in milliseconds.', 'performance-lab' ),
 					array( 'strong' => array() )
 				);
 				if ( ! perflab_server_timing_use_output_buffer() ) {
-					?>
-					<br>
```

Comment:
I think this line break in particular makes sense to keep, not because of long lines but because of the underlying content of the two parts split by it.

Reply:
Makes sense, let's go with that!